tests/test_core.py

```diff
@@ -6969,8 +6969,9 @@ def encode_utf8(data):
       assert seen_lines.issuperset([6, 7, 11, 12]), seen_lines
 
   @no_wasm2js('TODO: source maps in wasm2js')
-  @no_mac("needs investigation, started to fail on https://chromium-review.googlesource.com/c/emscripten-releases/+/2560546")
   def test_dwarf(self):
+    if not is_optimizing(self.emcc_args):
+      self.skipTest('https://github.com/emscripten-core/emscripten/issues/12903')
```

Comment:
It looks like this removes all test coverage in unoptimized builds, on all platforms?

I think we should still test unoptimized builds on non-mac, as those builds are pretty important.

Reply:
The wasm0 configuration is currently broken on all platforms, not just mac.

We only noticed on mac because wasm0 only runs there.

Reply:
See https://github.com/emscripten-core/emscripten/issues/12903 for details.